### DISCUSSION TITLE
PLZ Help me, I don't know how to start

### DISCUSSION THREAD

**Discussion by unix96** [JustFly1984/react-google-maps-api#2347] [Q&A]
Hello, I am writing this message because I desperately need help to start on my project. I saw the website but I don't understand how things work, I have used the react-google-maps but the react-google-maps-api looks so much better but I found it difficult to initiate. I just need somebody to show me the way at first. I feel so hopeless

**Comment by qb-rivkyb** [comment]
Why don't you show us what you did so far and then we can see how to help you. Read the readme and see if you can understand what you need to do if not I can try to help.

**Reply by unix96** [reply]
Yeah, the ReadMe Demo works, but I don't understand the concept each time I see new window.google and apparently I can use the method of Google Map JS, I had to change the library I used google-react-map, but I prefer to use this one, but I don't know how.
